low-level/sensors/impl/generic/node_hw.py

```diff
@@ -958,6 +1013,15 @@ def _parse_fan_info(self, index, date, _time, device_id, sensor_id,\
         if is_last:
             fan_info.update(fan_specific_data_dynamic)
 
+        if alert_type == "fault":
+            self.faulty_resources[sensor_name] = {
+                'status': original_event,
+                'event': original_status,
+                'resource': 'fan'
```

Comment:
Resource value should be 'Fan' instead of 'fan'.
Resource name is already defined one. Please match with this.
```
    TYPE_FAN = 'Fan'
```

Reply:
Done